src/XeroPHP/Application.php

```diff
@@ -445,4 +450,17 @@ public function delete(Remote\Model $object)
 
         return $object;
     }
+
+    public function requestCompleted($response, $info, $headers) {
+        foreach($this->requestCompletedCallbacks as $callback) {
+            if($callback instanceof \Closure) {
```

Comment:
I think we can get rid of this `instanceof` check. It opens it up to all types of callables (i.e. array based) and I think it is better to let PHP error out and say "You passed me something this can't handle" rather than just silently skipping it.

Thoughts?

Reply:
Gotcha, [this commit](https://github.com/calcinai/xero-php/pull/587/commits/598393ec0d13cd96ecad85cc3a28816b2f3e451e) fixes that. Thanks :)

Reply:
no worries. Cool PR - should come in handy for a lot of people I think.